modin/experimental/core/storage_formats/hdk/query_compiler.py

```diff
@@ -556,7 +556,8 @@ def concat(self, axis, other, **kwargs):
         return self.__constructor__(new_modin_frame)
 
     def drop(self, index=None, columns=None):
-        assert index is None, "Only column drop is supported"
+        if index is not None:
```

Comment:
Why does the coverage not detect that we get into this branch? Do we have a test for this?

Reply:
I've checked in debugger and this code is executed by the test_drop_row() test. Removing this code, causes the test to fail.

Reply:
ok, let's continue in this discussion ([link](https://github.com/modin-project/modin/pull/4937/files#r1029442629))